dashboard/models.py

```diff
@@ -217,10 +219,19 @@ class ProgramEnrollment(Model):
     """
     user = ForeignKey(User, on_delete=CASCADE)
     program = ForeignKey(Program, on_delete=CASCADE)
+    hash = CharField(max_length=32, null=False, unique=True)
```

Comment:
Can we talk about this? Is this the hash that you use for building the URL to the learner record? How will this work for non-FA programs?